packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -221,23 +222,42 @@ const handleOriginRequest = async ({
     s3Origin.path = `${basePath}/static-pages`;
     request.uri = pagePath.replace("pages", "");
     addS3HostHeader(request, normalisedS3DomainName);
+
     return request;
   }
 
   const tBeforePageRequire = now();
-  const page = require(`./${pagePath}`); // eslint-disable-line
+  let page = require(`./${pagePath}`); // eslint-disable-line
   const tAfterPageRequire = now();
 
   log("require JS execution time", tBeforePageRequire, tAfterPageRequire);
 
   const tBeforeSSR = now();
   const { req, res, responsePromise } = lambdaAtEdgeCompat(event.Records[0].cf);
-  page.render(req, res);
+  try {
+    // If page is _error.js, set status to 404 so _error.js will render a 404 page
+    if (pagePath === "pages/_error.js") {
+      res.statusCode = 404;
```

Comment:
Not completely sure why this works, I think the params might be slightly named incorrectly in case of `_error.js` page? Will need to look into the code more for the `render()` function in `_error.js`.

Reply:
`statusCode` follows the property name from NodeJS's HTTP module, which is followed by express, which in turn I think is followed by NextJS.

Reply:
I see, makes sense, thanks.